### PR TITLE
install and use cockroach benchstat in benchmarks

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

File: scripts/run_benchmarks.sh [hunk 1]
```diff
@@ -52,6 +52,9 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+# ensure we have the latest version of our benchstat fork
+go get -u github.com/cockroachdb/benchstat
```
Comment (mberhault) [inline scripts/run_benchmarks.sh]: I've been staying away from pulling things in those scripts, intentionally. I think we should keep it that way.

Comment (dt) [inline scripts/run_benchmarks.sh]: I couldn't find where `benchstat` was installed. Is it just one-time/by-hand?

Comment (mberhault) [inline scripts/run_benchmarks.sh]: yup. Ditto for terraform and a couple other things I use.
It's probably fine to get this, but it runs the risk of conflicting with the existing binary depending on the paths.

Comment (dt) [inline scripts/run_benchmarks.sh]: it is in fact supposed to replace the existing binary in PATH, as this is a superset (a fork which adds json output).
Do you just want to run that `go get` by hand and close this?

Comment (mberhault) [inline scripts/run_benchmarks.sh]: LGTM.

let's go ahead and leave it. it's probably a good idea to start pulling
repos.

**Pull request merged**